docs/configuration/settings.rst

```diff
@@ -60,6 +60,10 @@ Feature settings
 |                                                 |              | endpoint: ``/v1`` redirects to ``/v1/`` and ``/buckets/default/``        |
 |                                                 |              | to ``/buckets/default``. No redirections are made when turned off.       |
 +-------------------------------------------------+--------------+--------------------------------------------------------------------------+
+| kinto.heartbeat_timeout_seconds                 | ``2``        | The maximum duration of each heartbeat entry. Depending of the amount of |
```

Comment:
2 seconds seems very low. Since we're running in parallel I think we can use 10 seconds by default maybe ?